cmd/aries-agent-mobile/pkg/wrappers/command/aries.go

```diff
@@ -149,6 +171,56 @@ func populateHandlers(commands []command.Handler, pkgMap map[string]map[string]c
 	}
 }
 
+func (a *Aries) startNotificationListener() {
+	// listens for notifications
+	for notification := range a.notifications {
+		a.mutex.RLock()
+		// gets all handlers that were subscribed for the topic
+		for _, handlers := range a.subscribers[notification.Topic] {
+			// send the payload to the subscribers
+			for _, handler := range handlers {
+				if err := handler.Handle(notification.Topic, notification.Raw); err != nil {
+					logger.Errorf("notification listener: %v", err)
+				}
+			}
+		}
+		a.mutex.RUnlock()
+	}
+}
+
+// RegisterHandler registers a handler to process incoming notifications from the framework.
+// Handler is implemented by mobile apps.
+func (a *Aries) RegisterHandler(h api.Handler, topics string) string {
```

Comment:
`topics` can be a var-arg

Reply:
@llorllale As far as I remember we cannot use variadic arguments.  @ToJen Could you please confirm this?
e.g RegisterHandler(h api.Handler, topics ...string) is not possible to use, @ToJen correct?.

Reply:
yes we can only use simple args like `string`